src/main/java/org/spdx/library/model/ExternalDocumentRef.java

```diff
@@ -224,6 +224,16 @@ public ExternalDocumentRef setSpdxDocumentNamespace(String documentNamespace) th
 						public String getIndividualURI() {
 							return documentNamespace;
 						}
+						
+						@Override
+						public boolean equals(Object comp) {
+							return SimpleUriValue.isIndividualUriValueEquals(this, comp);
+						}
+
+						@Override
+						public int hashCode() {
+							return SimpleUriValue.getIndividualUriValueHash(this);
+						}
 			});
```

Comment:
Is there a reason for using an anonymous class instead of a `SimpleUriValue` instance now?

Reply:
I'm just using `SimpleUriValue` as a convenient place to refactor the common code for all `IndividualUriValue` `equals` and `hashCode` as static methods.  It would probably be clearer if I created a separate static helper class, but I didn't want to create another class with just those methods.